packages/dso-toolkit/components/Voorbeelden/20-Toepassingen/10-Aanvragen/activiteiten.config.yml

```diff
@@ -3,7 +3,8 @@ order: 40
 status: ready
 context:
   steps:
```

Comment:
is `steps` dan nog wel de meest juiste benaming? niet liever 'appheading' ?

Reply:
het wordt inderdaad `appheading` waarvan `steps` een onderdeel kan zijn